Autopilot/boardfiles/Src/main.c

```diff
@@ -130,14 +130,15 @@ int main(void)
 
   /* USER CODE BEGIN SysInit */
 
+  HAL_Delay(100);
+
```

Comment:
Why this delay?

Reply:
No clue... @Fberbari any comments? (git blame points you you lol)

It's not in the while loop so it's not too much of a concern for me, but if it's not needed we could scrap it